data/helpers.d/nodejs

```diff
@@ -98,7 +98,12 @@ ynh_install_nodejs () {
 	test -x /usr/bin/npm_n && mv /usr/bin/npm_n /usr/bin/npm
 
 	# Install the requested version of nodejs
-	n $nodejs_version
+	if [[ $uname =~ aarch64 ||  $uname =~ arm64]]
```

Comment:
I think you need to define `$uname` tho :P 

Probably `uname=$(uname -m)`

Reply:
of course ^^ I done it too quickly...